packages/muze-legend/src/styles.scss

```diff
@@ -60,6 +60,11 @@ $prefix: muze-;
     cursor: pointer;
     font-size: 12px;
 }
+
+.#{$prefix}legend-item-info, .#{$prefix}legend-title {
```

Comment:
We can't directly inherit the top class (i.e Muze-legend-box in this case) as it will also inherit other properties also of that class. have to create a font-family placeholder selector which can be later on inherited.

Reply:
modified the scss file accordingly